lib/README.md

```diff
@@ -1,13 +1,13 @@
-![Arrow Banner](https://github.com/Arrow-air/.github/raw/main/profile/assets/arrow_v2_twitter-banner_neu.png)
+![Aetheric Banner](https://github.com/Arrow-air/tf-github/raw/main/src/templates/doc-banner-services.png)
```

Comment:
should point to https://github.com/aetheric-oss/.github/blob/main/assets/readme-banner.png